flutter-for-react-native/index.md

```diff
@@ -0,0 +1,2411 @@
+  },
+);
+```
+For more information, including a list of Flutter `GestureDetector` callbacks,
+ see the [GestureDetector class](https://docs.flutter.io/flutter/widgets/GestureDetector-class.html#Properties).
+
+
+
+|Android|iOS|
+|:---:|:--:|
+|<img src="images/flutter_gestures_android.gif?raw=true" style="width:300px;" alt="Loading">|<img src="images/flutter_gestures_iOS.gif?raw=true" style="width:300px;" alt="Loading">|
+
+<br>
+
+## Making HTTP networking requests
+
+Fetching data from the internet is common for most apps. And in Flutter, the `http` package provides the simplest way to fetch data from the internet.
+
+### How do I fetch data from API calls?
+
+React Native provides the Fetch API for networking—you make a fetch request
+and then receive the response to get the data.
+
+<!-- skip -->
+```JavaScript
+// React Native
+_getIPAddress = () => {
+  fetch("https://httpbin.org/ip")
+    .then(response => response.json())
+    .then(responseJson => {
+      this.setState({ _ipAddress: responseJson.origin });
+    })
+    .catch(error => {
+      console.error(error);
+    });
+};
+```
+Flutter uses the `http` package. To install the `http` package, add it to the dependencies section of our pubspec.yaml.
+<!-- skip -->
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  http: <latest_version>
+```
+
+Flutter uses the [`dart:io`](https://docs.flutter.io/flutter/dart-io/dart-io-library.html) core HTTP support client. To create an HTTP Client, import `dart:io`.
+
+<!-- skip -->
+```Dart
+import 'dart:io';
+```
+
+The client supports the following HTTP operations: GET, POST, PUT, and DELETE.
+
+<!-- skip -->
+```Dart
+// Flutter
+final url = new Uri.https('httpbin.org', 'ip');
+final httpClient = new HttpClient();
+_getIPAddress() async {
+  var request = await httpClient.getUrl(url);
+  var response = await request.close();
+  var responseBody = await response.transform(utf8.decoder).join();
+  String ip = json.decode(responseBody)['origin'];
+  setState(() {
+    _ipAddress = ip;
+  });
+}
+```
+
+
+
+|Android|iOS|
+|:---:|:--:|
+|<img src="images/api_calls_android.gif?raw=true" style="width:300px;" alt="Loading">|<img src="images/api_calls_iOS.gif?raw=true" style="width:300px;" alt="Loading">|
+
+<br>
+
+## Form input
+
+Text fields allow users to type text into your app so they can be used to build
+forms, messaging apps, search experiences, and more. Flutter provides two
+core text field widgets: [TextField](https://docs.flutter.io/flutter/material/TextField-class.html) and [TextFormField](https://docs.flutter.io/flutter/material/TextFormField-class.html).
+
+### How do I use text field widgets?
+
+In React Native, to enter text you use a `TextInput` component to show a text
+input box and then use the callback to store the value in a variable.
+
+<!-- skip -->
+```JavaScript
+// React Native
+<TextInput
+  placeholder="Enter your Password"
+  onChangeText={password => this.setState({ password })}
+ />
+<Button title
```

Comment:
This is not the same sentence that appears in the staged version. There it says:

In Flutter, we can achieve this animation by using the Dismissible widget and nest the child widgets.

Reply:
I don't see this sentence in the file.